akka-actor/src/main/scala/akka/pattern/AskSupport.scala

```diff
@@ -261,18 +261,6 @@ trait ExplicitAskSupport {
 
 object AskableActorRef {
 
-  /**
-   * INTERNAL API: for binary compatibility
-   */
-  private[pattern] def ask$extension(actorRef: ActorRef, message: Any, timeout: Timeout): Future[Any] =
```

Comment:
Needs doublechecking/fixing

Reply:
It seems the naming of generated extension methods changed in RC1 (https://github.com/scala/scala/commit/f9879b63a87b1e3062c58abd9ce30e68d9fee8ca), causing the generated extension method for the protected 'for bincompat' to clash with our manual 'for bincompat' extension method.

Since the 'for bincompat' methods have not been exposed as API for a long time I think we can safely remove at least their 'manual' extension methods, and perhaps the methods themselves as well.